### PR TITLE
#6 patch article completed

### PR DESCRIPTION
**Pull request by nicolalockwood**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -32,3 +32,115 @@ describe('GET /api/topics', () => {
...
+			.get('/api/articles/1')
+			.expect(200)
+			.then((res) => {
+				expect(res.body.article).toEqual({
+					article_id: 1,
+					title: 'Living in the shadow of a great man',
+					topic: 'mitch',
+					author: 'butter_bridge',
+					body: 'I find this existence challenging',
+					created_at: expect.any(String),
```
Comment (dentednerd) [inline __tests__/app.test.js:48]: Nicely handled!

File: models/article.model.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		.query('SELECT votes FROM articles WHERE article_id = $1 ;', [id])
+		.then(({ rows }) => {
+			let currentVote = rows[0].votes;
+			let newVote = (currentVote += voteUpdateAmmount);
+			return newVote;
+		})
+		.then((newVote) => {
+			return db
+				.query(
+					'UPDATE articles SET votes =$1 WHERE article_id =$2 RETURNING *;',
```
Comment (dentednerd) [inline models/article.model.js:42]: All good, just watch the spaces around your = here

File: package.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
 		"supertest": "^6.2.2"
 	},
 	"dependencies": {
+		"@types/jest": "^27.4.1",
```
Comment (dentednerd) [inline package.json:32]: Oooh, where's this come from?

**Review by dentednerd — APPROVED** [review]

File: models/article.model.js (hunk 2 above)
Comment (nicolalockwood) [reply models/article.model.js:42]: Thank you so much will update accordingly :)

**Pull request merged**